proto/decentraland/sdk/components/material.proto

```diff
@@ -22,18 +22,18 @@ message PBMaterial {
     optional float alpha_test = 2; // default = 0.5. range value: from 0 to 1
     optional bool cast_shadows = 3; // default =  true
     optional decentraland.common.Color4 diffuse_color = 4; // default = white;
-    optional decentraland.common.TextureUnion alpha_texture = 5; // default = null
+    optional decentraland.common.TextureUnion alpha_texture = 5; // default = null. Note that tilling and offset properties are ignored for this texture.
   }
 
   message PbrMaterial {
     optional decentraland.common.TextureUnion texture = 1; // default = null
   
     optional float alpha_test = 2; // default = 0.5. range value: from 0 to 1
     optional bool cast_shadows = 3; // default =  true
-    // @deprecated Alpha textures are no longer supported on PBRMaterial and UnlitMaterial.alphaTexture should be used instead.
-    optional decentraland.common.TextureUnion alpha_texture = 4; // default = null
-    optional decentraland.common.TextureUnion emissive_texture = 5; // default = null
-    optional decentraland.common.TextureUnion bump_texture = 6; // default = null
+  
+    optional decentraland.common.TextureUnion alpha_texture = 4; // @deprecated Alpha textures are no longer supported on PBRMaterial and UnlitMaterial.alphaTexture should be used instead.
```

Comment:
the @deprecated comment works if you put it in the same line ?
You should see this message in the scene when you put the mouse over this prop, or in the typings file.
(Asking becuase i'm not sure how its being propagated)

Reply:
Thanks for sharing your concerns!
As far as I know (after reading the documentation and doing a brief research), these are just comments, they don't actually do anything and it's up to the interpreter for each language what they do with it. The generated code in the client will always have these fields.
It's actually recommended to actually remove the deprecated properties and just `reserve` the id numbers.
I'll let @leanmendoza provide some feedback on the subject.

Reply:
The `reserve` should be put after one or two minor SDK releases. (and previously communicated the breaking change). It's not a big deal because it's breaking only dead code (assuming this functionality has already passed away).

About the comment, I think both, same line and previous line, forward the comment to generated code in the SDK. It's worth to confirm.

Reply:
Btw, it's something that if it doesn't fit the forwarding needs, it's can be re-visited in the future without introducing breaking changes

Reply:
![image](https://github.com/user-attachments/assets/ffe6b54f-d50d-4a90-ba08-00341bca4f73)
Here's the generated code in the client